doc/chapter-files.asciidoc

```diff
@@ -7,8 +7,8 @@ which splits the files between the following locations:
 - _~/.local/share/newsboat/_
 - _~/.config/newsboat/_
 
-If these directories exist, Newsboat will use them, otherwise it will default
-to _~/.newsboat_.
+If the config directory exists, Newsboat will use XDG directories, otherwise it
+will default to _~/.newsboat_.
```

Comment:
I think adding the information from the commit message helps, too: "… When using XDG directories, Newsboat will create the data directory automatically, in case it is missing."

Reply:
Your version is even better! :-)